docs/diagrams/room/RoomClass.puml

```diff
@@ -0,0 +1,31 @@
+@startuml
+!include ../style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor LOGIC_COLOR_T4
+skinparam classBackgroundColor LOGIC_COLOR
```

Comment:
Consider using `MODEL_COLOR` instead of `LOGIC_COLOR` since this is part of the model component.

Reply:
Done, thanks for pointing that out